.releaserc.js

```diff
@@ -44,6 +44,7 @@ const releaseConfig = {
                 parserOpts
             }
         ],
+        '@semantic-release/npm',
```

Comment:
The default setting for `@semantic-release/npm` is to also publish to our npm registry.  We should set the `npmPublish` configuration option to false.

The actual publishing to npm happens as part of the release github workflow.

Reply:
Oh ok, good catch. I'll try it out on my fork and update later this afternoon.

Reply:
So I tried setting `npmPublish` to false on my fork (in https://github.com/dbirks/shepherd/commit/bc604397e741e8f1f13cc353357c824b1b725f7f), but it didn't publish a package (no 1.15.6 version [here](https://github.com/dbirks/shepherd/packages/1019243)). 

Looking at the Actions in the release.yml, I don't see anything else that would publish the package, so I think it's been semantic-release that's been doing it.

Here's some examples of it publishing in my fork: the [Release Action](https://github.com/dbirks/shepherd/runs/3769772989?check_suite_focus=true), and the commit it makes with the changelog, package.json, and package-lock bumped (https://github.com/dbirks/shepherd/commit/32ba69651fdd125a8141236ff66216f3095ead5c).

Let me know if you're seeing something I'm not seeing. :mag:

Reply:
Ah, right!

Then this should be fine.  I was mainly concerned that the version in Github would be behind the version in npm.

Reply:
Ok cool! :slightly_smiling_face: